app/lib/search/backend.dart

```diff
@@ -45,6 +45,21 @@ void registerSnapshotStorage(SnapshotStorage storage) =>
 SnapshotStorage get snapshotStorage =>
     ss.lookup(#_snapshotStorage) as SnapshotStorage;
 
+/// The [PackageIndex] for Dart SDK API.
+PackageIndex get dartSdkIndex => ss.lookup(#_dartSdkIndex) as PackageIndex;
+
+/// Register a new [PackageIndex] for Dart SDK API.
+void registerDartSdkIndex(PackageIndex index) =>
+    ss.register(#_dartSdkIndex, index);
+
+/// The [PackageIndex] registered in the current service scope.
+PackageIndex get packageIndex =>
```

Comment:
Can we explain that this is available everywhere?

Reply:
I assume it is the `packageIndex` for use in case search service is failing, we can fallback to this. So it's available on the frontend service. (Am I correct?)